src/wwwroot/services/auth.js

```diff
@@ -37,25 +37,41 @@
       isUrlAllowed: isUrlAllowed,
       getDefaultUrl: getDefaultUrl
     };
-
+    var loginSession = null;
     var decodedToken = null;
     var encodedToken = null;
```

Comment:
Do you think that it is possible to remove _decodedToken_ or at least _encodedToken_ and use _loginSession_ in place?

Reply:
I think that it is possible, maybe you should replace _decodedToken_ by _loginSession_

Reply:
I don't really sure about that, we need to define where will be stored the permission account info.